packages/server/src/helpers/session-middleware.js

```diff
@@ -1,7 +1,7 @@
 import expressSession from 'express-session'
 import { Session } from '../models/objection'
 
-import { HOURS, DAYS } from '@aragon/protocol-backend-shared/helpers/times'
+import { HOURS, DAYS } from '@aragon/protocol-backend-shared/build/helpers/times'
```

Comment:
This may be a dumb question, but when we update these other helpers to be in TS, would we be able to import the source files?

It seems a bit weird to have to do this.

Reply:
Yes I added .ts extension to babel, so you can import .ts files into .js

Reply:
One way to do it would be to define `main` and `types` in `package.json` like I did in ipfs pinner and then have all imports in a single `index.ts` file:
https://github.com/aragonone/ipfs-pinner/blob/development/packages/shared/src/index.ts

In this case we could import the same thing like this:
```js
import { times } from '@aragon/protocol-backend-shared
const { HOURS, DAYS } = times
```
I find this more verbose however and you also have to think about multiple imports/exports not just the source file directly.